stackslib/src/chainstate/nakamoto/coordinator/mod.rs

```diff
@@ -93,11 +110,13 @@ impl OnChainRewardSetProvider {
               "liquid_ustx" => liquid_ustx,
               "registered_addrs" => registered_addrs.len());
 
-        Ok(StacksChainState::make_reward_set(
-            threshold,
-            registered_addrs,
-            cur_epoch.epoch_id,
-        ))
+        let reward_set =
+            StacksChainState::make_reward_set(threshold, registered_addrs, cur_epoch.epoch_id);
+        if reward_set.signers.is_none() {
+            error!("FATAL: PoX reward set did not specify signer set in Nakamoto");
```

Comment:
Is this truly fatal?  If so, then should it panic?

Reply:
I think its as fatal as non-participation, so it should be treated the same.